editing_and_imputation/imputation/ratio_of_means/methodological_specification.md

```diff
@@ -118,16 +147,37 @@ types:
 * Forward Link Observation Count - Numeric
 * Backward Link Observation Count - Numeric
 * Construction Link Observation Count - Numeric
+* Forward Link Default Marker – Boolean
+* Backward Link Default Marker – Boolean
+* Construction Link Default Marker – Boolean
+
+If Responder Filtering is performed, then output records should also contain:
+
+* Post Filter Inclusion Marker Previous – Boolean – Nulls Allowed
+* Post Filter Inclusion Marker Target – Boolean – Nulls Allowed
+* Post Filter Inclusion Marker Next – Boolean – Nulls Allowed
+
+If weighted imputation is performed, then the output record should also
+contain:
+
+* Unweighted Forward Link - Numeric - Optional - Nulls Allowed
+* Unweighted Backward Link - Numeric - Optional - Nulls Allowed
+* Unweighted Construction Link - Numeric - Optional - Nulls Allowed
 
 Fields of type "Any" shall be of the same type as the corresponding input
 fields as the values shall be the same in both input and output records.
 
+Output records labelled 'Optional' shall only be populated in the ouput
+dataset if the corresponding functionality was used e.g., trimming.
```

Comment:
I would give different e.g wighted or response filter rather than `trimming`. Because trimming is not used in RoM as previously mentioned in this PR 'Due to its robust nature, it does not use any form of trimming or outliering.'